native~/Runtime/src/CesiumBingMapsRasterOverlayImpl.cpp

```diff
@@ -81,7 +81,7 @@ void CesiumBingMapsRasterOverlayImpl::AddToTileset(
       overlay.bingMapsKey().ToStlString(),
       mapStyle,
       "",
-      CesiumGeospatial::Ellipsoid::WGS84,
+      tileset.NativeImplementation().getTileset()->getEllipsoid(),
```

Comment:
Could replace this with `pTileset->getEllipsoid`, since `pTileset` has already been checked against null.